lib.js

```diff
@@ -0,0 +1,28 @@
+function computeTitle(title) {
+  const getting = browser.storage.sync.get({
+    prefix: '',
+    prefixGlue: '',
+  });
+
+  return getting.then((result) => {
+    const computedTitle = result.prefix
+      + ((result.prefix && title && result.prefixGlue) || '')
+      + (title || '');
+
+    return computedTitle ? `[${computedTitle}] ` : '';
+  });
+}
+
+function setWindowTitle(windowId) {
```

Comment:
It not only sets but restores so would restoreWindowTitle be a more precise name?

Reply:
ok